bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/contentfragment/ContentFragmentImpl.java

```diff
@@ -13,22 +13,21 @@
  ~ See the License for the specific language governing permissions and
  ~ limitations under the License.
  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
-package com.adobe.cq.wcm.core.components.extension.contentfragment.internal.models.v1;
-
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.Iterator;
-import java.util.LinkedHashMap;
-import java.util.LinkedList;
-import java.util.List;
-import java.util.Map;
-import javax.annotation.Nonnull;
-import javax.annotation.Nullable;
-import javax.annotation.PostConstruct;
-import javax.inject.Inject;
+package com.adobe.cq.wcm.core.components.internal.models.v1.contentfragment;
 
+import com.adobe.cq.dam.cfm.ContentElement;
```

Comment:
Trivial stuff: we normally organise imports so that first are `java.*`, then `javax.*`, then `org.*` and only then `com.*`. It would be nice if you could configure your IDE to do the same so that there are less line changes and they won't change back again when somebody touches these files.

Reply:
Consider it done (even though I'm not a fan of such convention).

Reply:
Thanks!

Reply:
I quickly checked the other classes to see what the standard is: seems you have empty lines between all the groups. This is what I used in my tmp-profile.